axelrod/strategies/axelrod_tournaments.py

```diff
@@ -29,6 +29,7 @@ class Davis(Player):
     def __init__(self, rounds_to_cooperate=10):
```

Comment:
It's the number of rounds_to_cooperate. We can update the docstring, but frankly few if any of the strategies explain their parameters, so I'd prefer to open up another issue for that.